pilot/pkg/config/coredatamodel/discovery.go

```diff
@@ -283,9 +433,9 @@ func convertServicePorts(ports []*networking.Port) model.PortList {
 	return out
 }
 
-// Services Not Supported
-func (d *MCPDiscovery) Services() ([]*model.Service, error) {
-	log.Warnf("Services %s", errUnsupported)
+// GetService Not Supported
+func (d *MCPDiscovery) GetService(hostname host.Name) (*model.Service, error) {
```

Comment:
Is this never called?

Reply:
GetService is not really needed in this discovery, the important one was the Services which was not initially implemented and external serviceDiscovery was returning synthetic service entries instead which was causing some really funky behaviour 😮